uber/site_sections/jobs.py

```diff
@@ -172,6 +177,15 @@ def staffers_by_job(self, session, id, message=''):
                                 .order_by(Attendee.full_name).all()
         }
 
+    def allStaffers(self, session, id, message=''):
+        job = session.job(id);
```

Comment:
``pep8`` is complaining that this line ends with a semicolon, which is why Travis says that the build failed.  Can you remove that and push out the new commit to this branch?  That will automatically update the Pull Request.